src/Helpers/async-pipe.ts

```diff
@@ -1,4 +1,86 @@
-type Func = (v?: any) => any | Promise<any>
+export function asyncPipe<V0 extends any, T1>(fn1: (x?: V0) => T1 | Promise<T1>): (x?: V0) => Promise<T1>
```

Comment:
Use `asyncPipe` as a name (following AirBnB's naming standard for files)

Reply:
will rename in follow up PR 👍